src/Tag/Hr.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Html\Tag;
+
+final class Hr extends VoidTag
```

Comment:
I don't think it is particularly useful since it is rarely used along with classes / attributes.

Reply:
But if the plan is to have [all tags from HTML5 spec](https://html.spec.whatwg.org/multipage/#toc-semantics)  then it's OK.

Reply:
HR removed